assets/queries/k8s/default_namespace/query.rego

```diff
@@ -19,14 +21,21 @@ CxPolicy[result] {
 CxPolicy[result] {
 	document := input.document[i]
 
+    checkKind(document.kind)
+
 	metadata = document.metadata
 	metadata.namespace == "default"
 
 	result := {
 		"documentId": input.document[i].id,
 		"issueType": "IncorrectValue",
-		"searchKey": sprintf("metadata.name=%s.namespace", [metadata.name]),
+		"searchKey": sprintf("metadata.name={{%s}}.namespace", [metadata.name]),
 		"keyExpectedValue": sprintf("Default namespace is not used in document[%d]", [i]),
 		"keyActualValue": sprintf("Default namespace is used in document[%d]", [i]),
 	}
 }
+
+checkKind(kind) {
```

Comment:
This function *checkKind* seems to be relevant to many Kubernetes queries. It might make sense to place it in the library.